scripts/airtable-test-helpers/create_scale_test_components.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+# import pyairtable as pyair
+from pyairtable import Api
+
+from unstructured.ingest.logger import logger
+
+SCALE_TEST_NUMBER_OF_RECORDS = 20_000
+
+token = os.environ["AIRTABLE_ACCESS_TOKEN_WRITE"]
+large_table_base_id = os.environ["LARGE_TABLE_BASE_ID"]
```

Comment:
is LARGE_TABLE_BASE_ID something you create and get from the airtable interface? can you include a note on how?

Reply:
Added comments on what these IDs represent and how to re-create their represented components in case they get lost 👍:
https://github.com/Unstructured-IO/unstructured/blob/68c754be426928350b85771400c402a0a7cf0eaa/scripts/airtable-test-helpers/create_scale_test_components.py#L10-L33

https://github.com/Unstructured-IO/unstructured/blob/68c754be426928350b85771400c402a0a7cf0eaa/scripts/airtable-test-helpers/component_ids.sh#L1-L27